events_structs.go

```diff
@@ -855,3 +858,69 @@ func (ev *EventStackChangedType) String() string {
 	d := ev.TypeData
 	return fmt.Sprintf("Stack changed from '%s' to %s", d.PreviousStackName, d.CurrentStackName)
 }
+
+// Database maintenance planned
+type EventPlanDatabaseMaintenancedTypeData struct {
+	AddonName     string `json:"addon_name"`
+	MaintenanceID string `json:"maintenance_id"`
+}
+
+type EventPlanDatabaseMaintenanceType struct {
+	Event
+	TypeData EventPlanDatabaseMaintenancedTypeData `json:"type_data"`
+}
+
+func (ev *EventPlanDatabaseMaintenanceType) String() string {
+	return fmt.Sprintf("A maintenance (ID: %s) has been scheduled on the %s database.", ev.TypeData.MaintenanceID, ev.TypeData.AddonName)
+}
+
+func (ev *EventPlanDatabaseMaintenanceType) Who() string {
+	if ev.TypeData.AddonName != "" {
+		return fmt.Sprintf("Database %s", ev.TypeData.AddonName)
+	}
```

Comment:
We want to use the user from the payload. Either remove the condition completely or change the condition to fallback, as you wish.

Reply:
You didn't take this comment into account

Reply:
Fixed